### PR TITLE
feat: Add support UICollectionReusableView

### LINKED ISSUE #23: Parse collectionReusableView
```xml
                            </collectionViewCell>
                        </cells>
                        <collectionReusableView key="sectionHeaderView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" insetsLayoutMarginsFromSafeArea="NO" id="cAz-kJ-Bga">
                            <rect key="frame" x="0.0" y="0.0" width="375" height="50"/>
                            <autoresizingMask key="autoresizingMask"/>
                        </collectionReusableView>
                        <collectionReusableView key="sectionFooterView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" insetsLayoutMarginsFromSafeArea="NO" id="Qgm-eI-CZy">
                            <rect key="frame" x="0.0" y="100" width="375" height="50"/>
                            <autoresizingMask key="autoresizingMask"/>
                        </collectionReusableView>
                        <connections>
```
only two ? (footer and header)
decode a list of it? or make two attributes

### PR DESCRIPTION
**Pull request by r-plus**
Hi devs.

This PR add support decode `UICollectionReusableView` to model.
I want to use it for recently added `reuse_identifier` rule in IBLinter.

Closes #23

### CODE REVIEW

**Comment by r-plus** [comment]
oh, reusable view can have two in storyboard. I'll try to support it.

**Comment by r-plus** [comment]
@kateinoigakukun @phimage okey, finish up. Could you review this?

File: Sources/Models/Views/ScrollViews/CollectionView.swift [hunk 1]
```diff
@@ -35,6 +35,7 @@ public struct CollectionView: IBDecodable, ViewProtocol, IBIdentifiable {
     public let connections: [AnyConnection]?
     public let variations: [Variation]?
     public let cells: [CollectionViewCell]?
+    public let collectionReusableView: [CollectionReusableView]?
```
Comment (phimage) [inline Sources/Models/Views/ScrollViews/CollectionView.swift]: ok you make the choice to decode as a list

Comment (phimage) [reply Sources/Models/Views/ScrollViews/CollectionView.swift]: if we want direct accessor we can make it with code
```swift
var sectionHeaderView: CollectionReusableView? {
   return collectionReusableView?.filter { $0.key == "sectionHeaderView" }
}
```
(not tested)

Comment (r-plus) [reply Sources/Models/Views/ScrollViews/CollectionView.swift]: yes, I’ll do it if you want.

Comment (phimage) [reply Sources/Models/Views/ScrollViews/CollectionView.swift]: yes if you can 😇

Comment (r-plus) [reply Sources/Models/Views/ScrollViews/CollectionView.swift]: alright, did it.
and rename collectionReusableView to collectionReusableView`s`.

**Comment by kateinoigakukun** [comment]
Sorry for keeping you waiting 🙇

**Comment by r-plus** [comment]
thx for merge :)

**Pull request merged**